package.json

```
@@ -1,20 +1,48 @@
 {
-  "name": "css-frameworks-ca",
-  "version": "1.0.0",
-  "description": "Test",
-  "main": "index.js",
-  "scripts": {
-    "build": "sass src/scss:dist/css",
-    "start": "sass --watch src/scss:dist/css & live-server"
-  },
-  "keywords": [],
-  "author": "Noroff",
-  "license": "ISC",
-  "devDependencies": {
-    "live-server": "^1.2.2",
-    "sass": "^1.54.8"
-  },
-  "dependencies": {
-    "bootstrap-dark-5": "^1.1.3"
-  }
+    "name": "css-frameworks-ca",
+    "version": "1.0.9",
+    "description": "Test",
+    "main": "index.js",
+    "scripts": {
+        "build": "sass src/scss:dist/css",
+        "start": "sass --watch src/scss:dist/css & live-server",
+        "prepare": "husky install",
+        "test": "npm run test-unit && npm run test-e2e-cli",
+        "test-unit": "jest",
+        "test-e2e": "cypress open",
+        "test-e2e-cli": "cypress run",
```

```suggestion
        "test-e2e-cli": "live-server & cypress run",
```
Since you already live-server in the project, we can make sure it's running before running the cypress tests.